packages/@guardian/source-foundations/src/colour/palette.ts

```diff
@@ -181,9 +182,10 @@ export const palette = {
 		200: colors.pinks[1],
 		300: colors.pinks[2],
 		400: colors.pinks[3],
-		500: colors.pinks[4],
-		600: colors.pinks[5],
-		800: colors.pinks[6],
+		450: colors.pinks[4],
```

Comment:
can you add the to story? https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/colour/palette.stories.mdx

Reply:
From what I can tell it's been added automatically, unless I'm missing something.

This the `palette` story for this branch: https://5e933197d94d9c002275a15e-qhzeudcqit.chromatic.com/?path=/story/packages-source-foundations-palette--page

<img width="448" alt="Screenshot 2022-07-14 at 10 57 49" src="https://user-images.githubusercontent.com/77005274/178967739-57c6b1ed-a46f-48c1-9665-38909aea8584.png">

Reply:
oh sorry joe! it's more sophisticated than i thought :)

Reply:
although if that's true, how come chromatic hasn't spotted it?

Reply:
Is it because when the story is snapshotted by Chromatic, the `lifestyle` palette is hidden?

<img width="448" alt="Screenshot 2022-07-14 at 13 14 58" src="https://user-images.githubusercontent.com/77005274/178980151-a58464c5-6f51-44fb-a0fa-0007b9c5707a.png">

Reply:
oh yeah of course! we should probably change that to show them all i guess?

Reply:
yeah i think so! I'll ticket that up and investigate - will be a longer page but defeats the object if we're hiding the palette